docs/java-rest/high-level/query-builders.asciidoc

```diff
@@ -81,6 +82,7 @@ This page lists all the available search queries with their corresponding `Query
 | {ref}/query-dsl-script-query.html[Script]                           | {query-ref}/ScriptQueryBuilder.html[ScriptQueryBuilder]                             | {query-ref}/QueryBuilders.html#scriptQuery-org.elasticsearch.script.Script-[QueryBuilders.scriptQuery()]
 | {ref}/query-dsl-percolate-query.html[Percolate]                     | {percolate-ref}/PercolateQueryBuilder.html[PercolateQueryBuilder]                   |
 | {ref}/query-dsl-wrapper-query.html[Wrapper]                         | {query-ref}/WrapperQueryBuilder.html[WrapperQueryBuilder]                           | {query-ref}/QueryBuilders.html#wrapperQuery-java.lang.String-[QueryBuilders.wrapperQuery()]
+| {ref}/query-dsl-rank-feature-query.html[Rank Feature]               | {mapper-extras-ref}/RankFeatureQuery.html[RankFeatureQueryBuilder]                  | {query-ref}/RankFeatureQueryBuilders.html#log-java.lang.String-[RankFeatureQueryBuilders.log()]
```

Comment:
Is there a reason that we point only to `log` method? As we have a dedicate query builder class should we organize in a similar way as PercolateQueryBuilder without supplying the method?
```
{ref}/query-dsl-percolate-query.html[Percolate] | {percolate-ref}/PercolateQueryBuilder.html[PercolateQueryBuilder]  

{ref}/query-dsl-rank-feature-query.html[Rank Feature]  | {mapper-extras-ref}/RankFeatureQuery.html[RankFeatureQueryBuilder]


```